plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/Core.melk

```diff
@@ -444,9 +444,11 @@ group partitioning {
     advanced option partition: Integer {
         label "Layout Partition"
         description
-            "Partition to which the node belongs to. If 'layoutPartitions' is true,
-             a pair of nodes with different partition indices is placed such that 
-             the node with lower index is placed to the left of the other node (with left-to-right layout direction)."
+            "Partition to which the node belongs to. If Layout Partitioning is true, a pair of nodes with different
```

Comment:
The latter part of this comment should (also) be part of the `partitioning.true` option, shouldn't it?